pulsar-broker/src/test/java/org/apache/pulsar/client/api/TopicReaderTest.java

```diff
@@ -487,6 +487,9 @@ public void testMessageAvailableAfterRestart() throws Exception {
 
     }
 
+    // Issue 3975: https://github.com/apache/pulsar/issues/3975
```

Comment:
We usually just write this info in the pull request - not the code.

Reply:
I saw it [here](https://github.com/apache/pulsar/blob/master/pulsar-broker/src/test/java/org/apache/pulsar/client/api/SimpleProducerConsumerTest.java#L2873) I thought it would be useful for bugs, removing it.